solvers/qpSpline/ContinuityConstraint.cpp

```diff
@@ -0,0 +1,26 @@
+#include "ContinuityConstraint.h"
+#include <cassert>
+
+ContinuityConstraint::ContinuityConstraint(int derivative_order, int first_spline_index, int second_spline_index) :
```

Comment:
Given that LinearConstraint doesn't exist yet in C++ and I don't want to block @rdeits, I'd opt for the latter.